libs/pika/coroutines/include/pika/coroutines/detail/context_posix.hpp

```diff
@@ -165,7 +165,7 @@ namespace pika::threads::coroutines {
         // some platforms need special preparation of the main thread
         struct prepare_main_thread
         {
-            constexpr prepare_main_thread() {}
+constexpr prepare_main_thread() {}
```

Comment:
To do: why is clang-format doing this?

Reply:
Changing the ridiculous `namespace pika { namespace threads { namespace coroutines { namespace detail { namespace posix { namespace pth {` to use a slightly less ridiculous, but still ridiculous, `namespace pika::threads::coroutines::detail::posix::pth {` seems to have convinced clang-format to format the file sensibly.